bin/build-drivers.md

```diff
@@ -36,10 +42,5 @@ clojure -M -m build-driver redshift
 Verify that a built driver looks correctly built.
 
 ```
-cd bin/verify-driver redshift
-clojure -M -m verify-driver redshift
-
-# or
-
-./bin/verify-driver redshift
+clojure -X:build:build/verify-driver :driver :mongo
```

Comment:
`bin/verify-driver` hasn't existed since like 2020, so another nice benefit of this PR is that while I was updating other dox I updated this too.